### ISSUE TITLE
string user variables include escaped double quotes

### ISSUE THREAD

**Issue by peterkleinct** [mailgun/mailgun-ruby#234]
I am experiencing a problem with user variables being sent through to Mailgun with additional escaped quote characters in the string. For example (taken from Mailgun logs):

```
  "user-variables": {
     "agent_name": "\"Test Contact\"",
     "agency_name": "\"Agency Name\"",
     "enquiry_id": "\"8903776A\"",
     "mail_type": "\"agent_enquiry\""
  }
```

The is causing problems because it is being sent back to me in a webhook in the same format and requires additional parsing to use correctly.  I thought this might be related to #200 , but is still occurring on 1.2.4.

The mail is being sent using Rails ActionMailer as follows:

```
mail(
  to: <...>,
  from: <...>,
  reply_to: <...>,
  subject: <...>
).tap do |message|
  message.mailgun_variables = {
     "agent_name" => @agent.full_name,
     "agency_name" => @agency.trading_name,
     "enquiry_id" => enquiry.id,
     "mail_type" => "agent_enquiry"
  }
```

Is there something wrong in the way I'm setting up the variables?

**Comment by lonroth** [comment]
@peterkleinct To avoid this, you can use:
```
        message.mailgun_headers = {
          "X-Mailgun-Variables" => '{ "score": "42" }'
        }
```

Hope it helps! 🙂

**Comment by Retttro** [comment]
Fixed in mailgun-ruby version 1.2.5. Closing.

**Comment by lonroth** [comment]
@Retttro Nice 👏
